### PR TITLE
Add posting date extractor

### PR DESCRIPTION
**Pull request by ZwEin27**

### CODE REVIEW

File: digPostingDateExtractor/date_manipulation.py
Comment (jasonslepicka) [inline digPostingDateExtractor/date_manipulation.py:145]: Any idea why some of these duplicated and commented out?

File: digPostingDateExtractor/date_manipulation.py
Comment (jasonslepicka) [inline digPostingDateExtractor/date_manipulation.py]: Remove commented out code

File: digPostingDateExtractor/date_manipulation.py
Comment (jasonslepicka) [inline digPostingDateExtractor/date_manipulation.py]: Remove commented out code

File: digPostingDateExtractor/date_manipulation.py
Comment (jasonslepicka) [inline digPostingDateExtractor/date_manipulation.py:399]: I feel like there should be a better way to handle parsing dates than chaining a bunch of exceptions.  Not necessary to address this right now, but I'll open an issue after merging the PR.

**Review by jasonslepicka — APPROVED** [review]

**Comment by jasonslepicka** [comment]
@ZwEin27 Can you just remove some of the commented out code?  I'll merge it after that.

**Comment by ZwEin27** [comment]
Yeah, sure, I will do that for comments, but for codes, I think we need to talk with Amandeep first, because this is his implementation,:)

**Pull request merged**
